triframe/phases/advisor.py

```diff
@@ -44,7 +44,7 @@ def advisor_fn_messages(state: triframeState) -> List[Message]:
         message = None
         if node.source == "actor_choice":
             message = Message(
-                role="user",
+                role="assistant",
```

Comment:
Again, this seems potentially like a bad idea to conflate the actor and advisor, though I'm very uncertain